flamedisx/likelihood.py

```diff
@@ -51,8 +51,8 @@ def __init__(
                 ty.Dict[str, pd.DataFrame]] = None,
             free_rates=None,
             batch_size=10,
-            max_sigma=3,
-            max_dim_size=70,
+            max_sigma=12,
```

Comment:
Sorry I am forgetting this @robertsjames why does max sigma need to be so large (12)?

Reply:
I think it does until the bounds are updated

Reply:
This is a result of adding in the S1/S2 noise, which makes the MLE of the final detected quanta subject to more fluctuations. This is back-propagated through the earlier bounds estimations and so they end up not covering enough without us making max_sigma larger. With any luck the MC bounds (coming next) will solve this problem...